### PR TITLE
add build task for online tutorial with all assets

### PR DESCRIPTION
**Pull request by arlimus**

### CODE REVIEW

File: www/tutorial/gulpfile.js [hunk 1]
```diff
@@ -19,11 +19,13 @@ gulp.task('webpack', function() {
 gulp.task('copy', function(){
   gulp.src([
       '*.png',
-      'app/**/*.css',
-      'app/**/*.html',
+      'app/**/*.{css,html,json}',
```
Comment (chris-rock) [inline www/tutorial/gulpfile.js:22]: 👍

**Review by chris-rock — APPROVED** [review]

File: tasks/www.rb [hunk 2]
```diff
@@ -16,10 +16,19 @@
 #
 
 require 'inquirer'
+require_relative 'shared.rb'
```
Comment (chris-rock) [inline tasks/www.rb:19]: missing

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @arlimus

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: tasks/www.rb (hunk 2 above)
Comment (arlimus) [reply tasks/www.rb:19]: Fixed thank you!!

**Pull request merged**